README.md

```diff
@@ -1,10 +1,6 @@
 # terraform-google-bootstrap
 
-This module was generated from [terraform-google-module-template](https://github.com/terraform-google-modules/terraform-google-module-template/), which by default generates a module that simply creates a GCS bucket. As the module develops, this README should be updated.
-
-The resources/services/activations/deletions that this module will create/trigger are:
-
-- Create a GCS bucket with the provided name
+The purpose of this module is to help bootstrap a GCP organization, creating all the required GCP resources & permissions to start using the Cloud Foundation Toolkit (CFT). For users who want to use Cloud Build & Cloud Source Repos for foundations code, there is also a submodule to help bootstrap all the required resources to do this.
 
 ## Usage
```

Comment:
We need a caveat section which explains that multiple applies may be required due to the dependency on the unmanaged Cloud Build service account.

Reply:
I believe I have resolved that issue now, I have been getting consistent results both locally and last couple of integration tests.